game/game.py

```diff
@@ -4,15 +4,26 @@
 
 def main():
     score = [0, 0]
+    choices = {'1': 'scissors',
+               '2': 'rock',
+               '3': 'paper'}
     while True:
         print(get_info())
-        winner = make_decision(get_user_choice(), get_comp_choice(1, 3))
+        user_choice = get_user_choice()
+        comp_choice = get_comp_choice(1, 3)
+        print_made_choices(user_choice, str(comp_choice), choices)
+        winner = make_decision(user_choice, comp_choice)
         score = update_score(score, winner)
-        print(winner)
-        print(score)
+        if winner != 'draw':
+            print("Winner: {0}".format(winner))
+        print('####Score####', '\nUser: {0}\nComputer: {1}'.format(*score))
```

Comment:
I didn't even know that we can use *score to split it into {1} and {2}, very nice